olive/workflows/run/run.py

```diff
@@ -76,6 +76,12 @@ def dependency_setup(config):
             "TorchTRTConversion": EXTRAS.get("torch-tensorrt"),
         },
     }
+    ORT_PACKAGE_TO_EP_MAPPING = {
+        "onnxruntime": "CPUExecutionProvider",
+        "onnxruntime-gpu": "CUDAExecutionProvider",
```

Comment:
How to handle the other GPU EP cases in this map?

Reply:
For now, it's only cuda for gpu. We could add an `ep` command line argument in the future if needed.

Reply:
This is just a heuristic. Since `onnxruntime-gpu` package should always have cuda ep. 

There's already some TODOs for figuring out the correct ort package https://github.com/microsoft/Olive/blob/main/olive/workflows/run/run.py#L99 

But that is orthogonal to this PR's change. This one is after the required package has been inferred. This only checks if the inferred package is present.

Reply:
Ah if this is only used for validation, that is fine.